packages/blazor-workspace/NimbleBlazor/README.md

```diff
@@ -132,18 +141,11 @@ To provide localized strings in a localized Blazor app:
 
 ### Using Nimble Blazor in a Blazor Hybrid app
 
-There is currently an [issue in ASP.NET Core](https://github.com/dotnet/aspnetcore/issues/42349) that prevents the necessary JavaScript that Nimble Blazor relies on from loading in a Blazor Hybrid application. The Demo.Hybrid project illustrates the current required steps for getting Nimble Blazor to work properly. This simply involves adding the script `NimbleBlazor.HybridWorkaround.js` in the `index.html` file in `wwwroot`:
+**Requirement:** Microsoft.AspNetCore.Components.WebView v8.0.4+
 
-wwwroot/index.html
-```html
-    ...
-    <script src="_framework/blazor.webview.js"></script>
-    <script src="_content/NimbleBlazor/nimble-components/all-components-bundle.min.js"></script>
-    <!-- This script is a workaround needed for Nimble Blazor to work in Blazor Hybrid.
-         See https://github.com/dotnet/aspnetcore/issues/42349 -->
-    <script src="_content/NimbleBlazor/NimbleBlazor.HybridWorkaround.js"></script>
-</body>
-```
+This updated WebView package include a fix so that the JavaScript initialization code that Nimble/Spright Blazor uses gets loaded correctly.
+Note that if using the Microsoft.AspNetCore.Components.WebView.Wpf package, it only specifies a dependency of Microsoft.AspNetCore.Components.WebView v8.0+, so you may to add need an explicit dependency on Microsoft.AspNetCore.Components.WebView to ensure a recent enough version is included.
+The Demo.Hybrid project in the Blazor solution illustrates this setup.
```

Comment:
put backticks / inline code style around the package names, i.e. `Microsoft.AspNetCore.Components.WebView`

Reply:
Done